### ISSUE TITLE
Your code is inconsistent with the model described in the paper

### ISSUE THREAD

**Issue by S-HeXing** [amri369/Pytorch-Iternet#5]
1.The miniunet in this paper is to define a single reuse, rather than multiple miniunets。
2.The final loss function is the sum of the outputs of multiple unet modules, not just the final miniunet loss

**Comment by amri369** [comment]
You are absolutely right. It's fixed in the last commit.
